src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -0,0 +1,79 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { TableComparatorService } from '../../../services/table-comparator.service';
+import { SortBy, SortOrder } from '../../../types/sort-properties';
+
+/**
+ * Column data for sortable table
+ */
+export interface ColumnData {
+  header: string;
+  headerToolTip?: string;
+  sortBy?: SortBy; // Don't provide this field if you don't want this column to be sortable
```

Comment:
```suggestion
  sortBy?: SortBy; // optional if the column is not sortable
```

I would say comment like this is better :P

Reply:
Yep much more concise